dialogflow/update_intent_test.py

```diff
@@ -33,7 +33,7 @@ def create_intent(project_id):
 
     intent = Intent()
 
-    intent.display_name = "fake_intent"
+    intent.display_name = "fake_intent_{}".format(uuid.uuid4())
```

Comment:
```suggestion
    intent.display_name = f"fake_intent_{uuid.uuid4()}"
```

Reply:
I'm also in favor of this suggested change.